src/Containers/OhmmsPETE/OhmmsMatrix.h

```diff
@@ -149,17 +149,17 @@ class Matrix
    *  Hacky but so is just making the matrix n x (n + padding) to handle row alignment.
    *  This is unoptimized.
    */
-  template<typename CONTAINER>
-  void assignUpperRight(const CONTAINER& other)
+  template<class T_FROM, typename ALLOC_FROM>
+  void assignUpperLeft(const Matrix<T_FROM, ALLOC_FROM>& from)
   {
-    auto& this_ref = *this;
-    size_t cols = std::max(this_ref.cols(), other.cols());
```

Comment:
Actually I was surprised that std::max didn't trigger segfault in psiMinv.assignUpperRight(psiMinv_FP) when sizes don't match.